runtime/src/main/java/org/corfudb/runtime/exceptions/WorkflowResultUnknownException.java

```diff
@@ -0,0 +1,10 @@
+package org.corfudb.runtime.exceptions;
+
+/**
+ * An exception that is thrown when the result of a workflow is unknown
+ * @author Maithem
+ */
+public class WorkflowResultUnknownException extends RuntimeException {
+    public WorkflowResultUnknownException() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)